src/main/scala/com/codacy/client/bitbucket/client/BitbucketClientBase.scala

```diff
@@ -256,7 +256,7 @@ abstract class BitbucketClientBase(val client: WSClient, credentials: Credential
         val statusMessage =
           s"""
              |Failed request:
-             |
+             |${error.toString}
```

Comment:
`.toString` is automatically called by the `s""` interpolator.
```suggestion
             |$error
```
If you prefer this way, it's fine either :)